articles/Installation and Upgrading/install-from-config-file.md

```diff
@@ -31,9 +34,11 @@ Example:
 ```bash
 # Normal CLI setup with detailed logs
 sudo -u www-data php tao/scripts/taoSetup.php /var/www/path/to/your/configFile.json -vvv
+php tao/scripts/taoSetup.php /path/to/your/configFile.json -vvv
```

Comment:
I think it's enough with the Windows difference above, repeating the command twice can be confusing